android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -379,12 +379,15 @@ public CameraType getCameraType() {
     /**
      * Find the geographic coordinates corresponding to the given position on screen
      * @param screenPosition Position in pixels from the top-left corner of the map area
-     * @return LngLat corresponding to the given point
+     * @return LngLat corresponding to the given point, or null if the screen position
+     * does not intersect a geographic location.
```

Comment:
We can mention "or null if the screen position does not intersect a geographic location, which can happen at higher tilt angles".